storage/client/src/main/java/com/azure/storage/blob/AppendBlobClientBuilder.java

```diff
@@ -60,20 +73,30 @@ public AppendBlobClientBuilder() {
         policies = new ArrayList<>();
     }
 
-    /**
-     * Constructs an instance of AppendBlobAsyncClient based on the configurations stored in the appendBlobClientBuilder.
-     * @return a new client instance
-     */
-    private AzureBlobStorageImpl buildImpl() {
+    private AzureBlobStorageBuilder buildImpl() {
```

Comment:
This builder also needs to accept a secondary endpoint, and we either need a customer Storage policy to retry against secondaries, or the Core retry policy needs to support it.

Reply:
The retry policy and options are being migrated.